src/environments/array.js

```diff
@@ -76,6 +93,19 @@ function parseArray(
                 throw new ParseError(`Failed to parse function after ${next}`);
             }
             rowGaps.push(cr.value.size);
+
+            possibleHLine = parser.nextToken.text;
+            while (/\s/.test(possibleHLine)) {
+                parser.consume();
+                possibleHLine = parser.nextToken.text;
+            }
+            if (possibleHLine === "\\hline") {
+                hlines.push(true);
```

Comment:
Can `group.value.addHLines = true;` be done here?

Reply:
@ylemkimon Stand by. I've got a code cleanup commit coming in about an hour. It will change this section of the code.